src/config_test.js

```diff
@@ -39,6 +39,14 @@ module.exports = {
         url = config.moduleUrl("ace/ext/textarea");
         assert.equal(url, "a/b/ext-textarea.js");
         config.set("packaged", false);
+
+        var callback = () => Promise.resolve("success");
```

Comment:
Add `/**global Promise*/` comment to disable the warning.

Reply:
Done!

Reply:
Seems, didn't helped, I could use some mock construction here to avoid this